### PR TITLE
more lingo words

### PR DESCRIPTION
**Pull request by twishasaraiya**
Issue#4: I have added 20 new lingo terms

### CODE REVIEW

File: src/components/words.js [hunk 1]
```diff
@@ -17,4 +17,25 @@ export default [
   {word: "Faccha", hint:"The noobs of KGP"},
   {word: "Frustapa", hint:"Anxiety and annoyance for a KGPian"},
   {word: "Bhajan", hint:"60% of DC searches"},
+  {word: "Funda", hint:"the basic principle behind something"},
+  {word: "Geez", hint:"eating someone's stuff as of its own"},
+  {word: "Load", hint:"To Have Tension"},
+  {word: "Fakka", hint:"An F-grade in any subject"},
+  {word: "Lodu", hint:"A person who takes lot of load"},
+  {word: "Banda", hint:"Any Boy"},
+  {word: "Matka", hint:"Any M. Tech. male student"},
```
Comment (djbarnwal) [inline src/components/words.js]: The word Matka is already present.

File: src/components/words.js [hunk 2]
```diff
@@ -17,4 +17,25 @@ export default [
...
+  {word: "Archi", hint:"The Architecture Department"},
+  {word: "Taapna", hint:" Copying assignments or examination answer-scripts from any person"},
+  {word: "Makhana", hint:"To spoil a job"},
+  {word: "Stud", hint:"Any student who is good in academics as well as other fields like sports"},
+  {word: "Bandi", hint:"Any girl"},
+  {word: "Batti", hint:"The Electrical Engineering Department"},
+  {word: "Hathora", hint:"The Mechanical Engineering Department"},
+  {word: "Junta", hint:"The human population"},
+  {word: "Chhagi", hint:"A C.G.P.A. in between 6 and 7"},
```
Comment (djbarnwal) [inline src/components/words.js]: Chhagi is already present in the list (although with a different spelling)

**Review by djbarnwal — CHANGES_REQUESTED** [review]
Hi @twishasaraiya  Firstly, thanks a lot for sending a PR 😄  Just a few minor changes are needed. There are some redundant words in the list which has to be removed.

**Comment by twishasaraiya** [comment]
ok I will change them

**Comment by djbarnwal** [comment]
@twishasaraiya  Have you closed this PR by mistake? This looks fine now and was ready to be merged.

**Comment by twishasaraiya** [comment]
Yes I closed it by mistake
I have made a new pull request

**Pull request closed without merging**